### PR TITLE
Tc006 aziza has been completed

### PR DESCRIPTION
**Pull request by AzizaMir26**
cool code here
cool code there

### CODE REVIEW

**Comment by AzizaMir26** [comment]
cool

**Comment by IgorL1991** [comment]
Test

File: src/aziza/TestCase006.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package aziza;
+
+public class TestCase006 {
+
+    //cool code
+    //all ready to review
+
```
Comment (IgorL1991) [inline src/aziza/TestCase006.java:7]: wow ! nice code

**Review by IgorL1991 — APPROVED** [review]

**Comment by IrynM** [comment]
am i in right place?

**Comment by AzizaMir26** [comment]
ok

**Pull request merged**